profiler/internal/pprofutils/pprofutils.go

```diff
@@ -0,0 +1,12 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package pprofutils
```

Comment:
It might be worth slightly documenting this package and explaining that it's only used in tests. Perhaps take a quick glance over the godoc rendering to make sure it's ~user~dev-friendly

Reply:
27c45c3